sdk/storage/Azure.Storage.Files.Shares/src/Sas/ShareSasBuilder.cs

```diff
@@ -152,7 +152,7 @@ public ShareSasBuilder()
         /// class to create a Blob Service Sas.
```

Comment:
Looks like there's another problem here - this should be Azure Files, not Blob Service.

Reply:
```suggestion
        /// class to create an Azure Files SAS.
```